svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -1028,6 +1028,8 @@ namespace SVF
 {
 const std::string SVFValue::toString() const
 {
+    if (strstr(this->getName().c_str(), "(SVFExternaCalllValue)") != NULL)
```

Comment:
This is very ad-hoc. You will need to add a string to SVFValue if it is constructed by ExtAPI?